src/plugins/intel_gpu/tests/unit/test_cases/loop_gpu_test.cpp

```diff
@@ -296,7 +297,7 @@ void test_loop_gpu_basic_concat_nested(bool is_caching_test)
     // set inner loop body
     /////////////////////////////////
     topology inner_loop_body(
```

Comment:
random spot: Could you add a functional test where the loop is created from ov::TensorIterator w/ -1 iter count?

Reply:
I will update functional tests for tensor iterator with -1 when I update new PR to support LSTMSequence w/ -1 seq_length